app/src/main/kotlin/at/bitfire/davdroid/ui/AppSettingsModel.kt

```diff
@@ -107,4 +113,61 @@ class AppSettingsModel @Inject constructor(
 
     val pushEndpoint = preference.unifiedPushEndpointFlow()
 
+    private val _pushDistributor = MutableStateFlow<String?>(null)
+    val pushDistributor get() = _pushDistributor
```

Comment:
Are these `get()`s and `stateIn`s necessary? As far as I remember from https://developer.android.com/topic/architecture/ui-layer/state-production#stateflow, the `asStateFlow()` just makes the `MutableStateFlow` a non-mutable `StateFlow`.

It is however already a `StateFlow`, so we don't need to call `stateIn`, which is used to convert a generic `Flow` to a `StateFlow`.

Reply:
True, not necessary